queries/special.py

```diff
@@ -1125,8 +1161,18 @@ def _play_film(qs: str, q: Query) -> AnswerType:
     "hvað er leyndarmál lífsins": _MEANING_OF_LIFE,
     "hvert er leyndarmál lífsins": _MEANING_OF_LIFE,
     "hver er sannleikurinn": _MEANING_OF_LIFE,
-    "hvað er 42": {"answer": "Sex sinnum sjö"},  # :)
-    "hvað meinarðu með 42": {"answer": "Sex sinnum sjö"},
+    "hvað er 42": {"answer": "Sex sinnum níu"},  # :)
```

Comment:
*hóst* *hóst*

Reply:
I misremembered Deep Thought's response in HHG. Apparently it's using a tridecimal system ;)